lib/ChangeDueDateDialog/LoanList.js

```diff
@@ -76,6 +76,7 @@ const LoanList = (props) => {
       columnMapping={{
         selected: <Checkbox
           name="selected-all"
+          aria-label="Select all"
```

Comment:
`Select all` needs to be a `<FormattedMessage />` instead of a hard-coded string.